webapp/sources/rudder/rudder-web/src/main/style/rudder/rudder-menu.scss

```diff
@@ -480,23 +483,95 @@ header.main-header .logo-lg img{
   list-style: none;
   overflow-x: hidden
 }
-.navbar-nav .notifications-menu>.dropdown-menu > li a,
 .navbar-nav>.tasks-menu>.dropdown-menu > li > a {
   display: block;
   white-space: nowrap;
-  border-bottom: 1px solid #f4f4f4
-}
-.navbar-nav .notifications-menu>.dropdown-menu > li a:hover,
-.navbar-nav>.tasks-menu>.dropdown-menu > li > a:hover {
-  background: $navbar-hover-color;
-  text-decoration: none
+  border-bottom: 1px solid #f4f4f4;
 }
-.navbar-nav .notifications-menu>.dropdown-menu > li a {
-  color: #444;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  white-space: nowrap;
-  padding: 10px
+.navbar-nav .notifications-menu>.dropdown-menu {
+  &>li a {
+    display: block;
+    white-space: nowrap;
+
+    color: #444;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    padding: 5px 10px;
+    pointer-events: none;
+  }
+
+  &>li {
+    .help-block {
+      font-size:80%;
+    }
+    #deployment-start .help-block {
+      margin:5px 0 0 10px;
+    }
+  }
+
+  .status-card {
+    button.status-card-title {
+      display: flex;
+      align-items: baseline;
+      font-size: 1.2rem;
+      text-align: left;
+      border: none;
+      &:hover,
+      &:active {
+        box-shadow: none;
+        outline: none !important;
+      }
+      .status-card-title-caret {
+        font-size: $h5-font-size;
+        transform: scale(1.1);
+        margin-right: .5rem !important;
+        color: $secondary;
+      }
+      &[aria-expanded="false"] > .status-card-title-caret {
+        @include caret(end);
+        &::after {
+          vertical-align: 0.05em;
+          margin-right: 0.2em;
+          transform: scaleY(0.9);
+          transition: ease;
+        }
+      }
+      &[aria-expanded="true"] > .status-card-title-caret {
+        @include caret(down);
+        &::after {
+          vertical-align: 0.1em;
+          transform: translate(-0.25rem) scaleY(0.9);
+          transition: ease;
+        }
+      }
+    }
+  }
+
+  button.regeneratePolicies,button.generationDetails {
+    pointer-events: all;
+    width: 100%;
+    display: flex;
+    padding: 0 26px;
+    color:#DA291C !important;
+  
+    &:hover {
+      background-color: #DA291C !important;
+      color: #fff !important;
+    }
+  } 
+
+  // this css is only used in other menus (api-authorization plugin) 
+  // and need to be more specific than just "a" tag
+  :not(.policy-generation)>li a {
+    border-bottom: 1px solid #f4f4f4;
+    padding: 10px;
+
+    &:hover {
+      background: $navbar-hover-color;
+      text-decoration: none
+    }
+  }
```

Comment:
Refactoring in the classes is keeping compatibility with the plugin, this shows how the old rendering differs from the current one.
The `:not(.policy-generation)` selector is there because now everything else than the `li a` within the `policy-generation` dropdown is "non-standard" i.e. later we should add specific classes in the plugin